discord/guild_const.go

```diff
@@ -49,38 +53,85 @@ const (
 	Banner GuildFeature = "BANNER"
 )
 
-type ExplicitFilter uint8
+// ExplicitFilter is the explicit content filter level of a guild.
+type ExplicitFilter nullable.Enum
 
-const (
-	NoContentFilter ExplicitFilter = iota
-	MembersWithoutRoles
-	AllMembers
+var (
+	// NullExplicitFilter serialized to JSON null.
+	// This should only be used on nullable fields.
+	NullExplicitFilter ExplicitFilter = nullable.EnumNull
+	// NoContentFilter disables content filtering for the guild.
+	NoContentFilter ExplicitFilter = 0
+	// MembersWithoutRoles filters only members without roles.
+	MembersWithoutRoles ExplicitFilter = 1
+	// AllMembers enables content filtering for all members.
+	AllMembers ExplicitFilter = 2
 )
 
-type Notification uint8
+func (f *ExplicitFilter) UnmarshalJSON(b []byte) error {
+	i, err := nullable.EnumFromJSON(b)
+	*f = ExplicitFilter(i)
 
-const (
-	AllMessages Notification = iota
-	OnlyMentions
+	return err
+}
+
+func (f ExplicitFilter) MarshalJSON() ([]byte, error) {
+	return nullable.EnumToJSON(nullable.Enum(f)), nil
+}
+
+// Notification is the default message notification level of a guild.
+type Notification nullable.Enum
+
+var (
+	// NullNotification serialized to JSON null.
+	// This should only be used on nullable fields.
+	NullNotification Notification = nullable.EnumNull
+	// AllMessages sends notifications for all messages.
+	AllMessages Notification = 0
+	// OnlyMentions sends notifications only on mention.
+	OnlyMentions Notification = 1
 )
 
-type Verification uint8
+func (n *Notification) UnmarshalJSON(b []byte) error {
+	i, err := nullable.EnumFromJSON(b)
+	*n = Notification(i)
 
-const (
-	NoVerification Verification = iota
+	return err
+}
+
+func (n Notification) MarshalJSON() ([]byte, error) { return nullable.EnumToJSON(nullable.Enum(n)), nil }
+
+// Verification is the verification level required for a guild.
+type Verification nullable.Enum
+
+var (
```

Comment:
Refer to above comment.

Reply:
Again the `ModifyGuild` payload allows this type to either be `null` or omitted.